packages/ember-glimmer/lib/environment.js

```diff
@@ -62,6 +62,29 @@ function wrapClassAttribute(args) {
   return args;
 }
 
+function wrapClassBindingAttribute(args) {
+  let hasClassBinding = args.named.has('classBinding');
+
+  if (hasClassBinding) {
+    let { value , type } = args.named.at('classBinding');
+    if (type === 'value') {
+      let [ prop, truthy, falsy ] = value.split(':');
+      if (prop === '') {
+        let syntax = HelperSyntax.fromSpec(['helper', ['-class'], [truthy], null]);
+        args.named.add('class', syntax);
```

Comment:
So if we get rid of the classBinding param, the deprecation doesn't happen. We can obviously add a deprecation here, but not sure it's worth it for a feature that's on it's way out anyways.